var/spack/repos/builtin/packages/dla-future/package.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
+    """DLA-Future library: Distributed Linear Algebra with Future"""
+
+    homepage = "https://github.com/eth-cscs/DLA-Future"
+    url = "https://github.com/eth-cscs/DLA-Future/archive/v0.0.0.tar.gz"
+    git = "https://github.com/eth-cscs/DLA-Future.git"
+    maintainers = ["rasolca", "albestro", "msimberg", "aurianer"]
+
+    version("0.1.0", sha256="f7ffcde22edabb3dc24a624e2888f98829ee526da384cd752b2b271c731ca9b1")
+    version("develop", branch="master")
```

Comment:
The branch name should match the version.  There is no longer any reason to not have them the same.

(See https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison.)

Reply:
Fixed in https://github.com/spack/spack/pull/38224/commits/f700b79c205361607915ffbd7e2a911117635134